packages/smooth_app/lib/pages/scan/scan_header.dart

```diff
@@ -50,48 +51,66 @@ class _ScanHeaderState extends State<ScanHeader> {
                   constraints: BoxConstraints(
                     maxWidth: constraints.maxWidth * 0.4,
                   ),
-                  child: ElevatedButton.icon(
-                    style: buttonStyle,
-                    icon: const Icon(Icons.cancel_outlined),
-                    onPressed: model.clearScanSession,
-                    label: Text(appLocalizations.clear),
+                  child: Tooltip(
+                    message: appLocalizations.scan_header_clear_button_tooltip,
+                    child: ElevatedButton.icon(
+                      style: buttonStyle,
+                      icon: const Icon(Icons.clear_all),
+                      onPressed: model.clearScanSession,
+                      label: Text(appLocalizations.clear),
+                    ),
                   ),
                 ),
                 ConstrainedBox(
                   constraints: BoxConstraints(
                     maxWidth: constraints.maxWidth * 0.6,
                   ),
-                  child: ElevatedButton.icon(
-                    style: buttonStyle,
-                    icon:
-                        const Icon(RankingFloatingActionButton.rankingIconData),
-                    onPressed: () async {
-                      final ContinuousScanModel model =
-                          context.read<ContinuousScanModel>();
-                      await model.refreshProductList();
-                      if (!mounted) {
-                        return;
-                      }
-                      await Navigator.push<void>(
-                        context,
-                        MaterialPageRoute<void>(
-                          builder: (BuildContext context) =>
-                              PersonalizedRankingPage(
-                            barcodes: model.productList.barcodes,
-                            title: ProductQueryPageHelper.getProductListLabel(
-                              model.productList,
-                              context,
+                  child: Tooltip(
+                    message: compareFeatureAvailable
+                        ? appLocalizations
+                            .scan_header_compare_button_valid_state_tooltip
+                        : appLocalizations
+                            .scan_header_compare_button_invalid_state_tooltip,
+                    child: AnimatedOpacity(
+                      opacity: compareFeatureAvailable ? 1.0 : 0.5,
+                      duration: SmoothAnimationsDuration.brief,
+                      child: ElevatedButton.icon(
+                        style: buttonStyle,
+                        icon: const Icon(Icons.compare_arrows),
```

Comment:
Please change `RankingFloatingActionButton.rankingIconData` instead, for consistency: the same icon for the same action.

Reply:
The reason I've changed the icon here, is because the `RankingFloatingActionButton.rankingIconData` is a trophy, and in personal search, this is the right icon to use :/

Reply:
What would be the best option according to you?
To differentiate the 🏆 and ↔️?